dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/monitoring/DynomiteManagerMetricsTask.java

```diff
@@ -0,0 +1,31 @@
+package com.netflix.dynomitemanager.monitoring;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.netflix.nfsidecar.scheduler.SimpleTimer;
+import com.netflix.nfsidecar.scheduler.Task;
+import com.netflix.nfsidecar.scheduler.TaskTimer;
+
+public class DynomiteManagerMetricsTask extends Task {
+    private static final Logger Logger = LoggerFactory.getLogger(DynomiteManagerMetricsTask.class);
+    public static final String TaskName = "Dynomite-Manager-Task";
+
+
+	@Override
+	public void execute() throws Exception {
```

Comment:
what is the purpose of this task?

Reply:
This is the work in progress piece. The task would execute metrics related to DM only. I am following this approach to capture REST failures in the metrics.